docs_src/microservices/application/available/AvailableAppServices.md

```diff
@@ -4,6 +4,6 @@ The following table lists the available EdgeX Application Services:
 
 | Repository                               | Status | Comments                                                                                                                                                                       | Documentation                                                |
 | ------------------------------------------------------------ | ------ |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ------------------------------------------------------------ |
-| [app-service-configurable](https://github.com/edgexfoundry/app-service-configurable/tree/{{version}}) | Active | App Service which provides configurable function pipelines capability for built-in pipeline functions                                                                          | [app-service-configurable docs](./AppServiceConfigurable.md) |
-| [app-rfid-llrp-inventory](https://github.com/edgexfoundry/app-rfid-llrp-inventory/tree/{{version}}) | Active | App Service which generate Inventory movement Events from raw LLRP events produced by [device-rfid-llrp](https://github.com/edgexfoundry/device-rfid-llrp-go/tree/{{version}}) | [app-rfid-llrp-inventory docs](./AppLLRPInventory.md)        |
-| [app-record-replay](https://github.com/edgexfoundry/app-record-replay/tree/{{version}}) | Active | App Service for Development/Testing with capability to Record and Replay EdgeX Events                                                                                          | [app-record-replay docs](./AppRecordReplay.md)               |
+| [app-service-configurable](https://github.com/edgexfoundry/app-service-configurable/tree/{{edgexversion}}) | Active | App Service which provides configurable function pipelines capability for built-in pipeline functions                                                                          | [app-service-configurable docs](./AppServiceConfigurable.md) |
+| [app-rfid-llrp-inventory](https://github.com/edgexfoundry/app-rfid-llrp-inventory/tree/{{edgexversion}}) | Active | App Service which generate Inventory movement Events from raw LLRP events produced by [device-rfid-llrp](https://github.com/edgexfoundry/device-rfid-llrp-go/tree/{{edgexversion}}) | [app-rfid-llrp-inventory docs](./AppLLRPInventory.md)        |
+| [app-record-replay](https://github.com/edgexfoundry/app-record-replay/tree/{{edgexversionmain}}) | Active | App Service for Development/Testing with capability to Record and Replay EdgeX Events                                                                                          | [app-record-replay docs](./AppRecordReplay.md)               |
```

Comment:
This is temporarily `main` until we release since there isn't a 3.0  tag on the new repo.
No need to create an macro for it.
```suggestion
| [app-record-replay](https://github.com/edgexfoundry/app-record-replay/tree/main) | Active | App Service for Development/Testing with capability to Record and Replay EdgeX Events                                                                                          | [app-record-replay docs](./AppRecordReplay.md)               |
```